public/app/features/alerting/unified/components/Expression.test.tsx

```diff
@@ -51,11 +51,11 @@ const rulesSource = {
 };
 
 describe('Expression', () => {
-  it('Should not allow to edit the text in the editor', () => {
+  it('Should not allow to edit the text in the editor', async () => {
```

Comment:
@gillesdemey are you able to look into whether this test makes sense anymore? as far as i can tell, there's nothing stopping the editor from being editable. so now when the `await` is added and it actually types the text correctly, this test fails because it really does type `something else` into the editor, so line 61 fails 😅

Reply:
Oh actually it looks like we don't need this test anymore! We removed the `readOnly` mode in https://github.com/grafana/grafana/pull/57839 it seems 👍

Reply:
the only thing i'm worried about is this comment at the bottom of that PR:
> The issue happened when the slate-react's Editor had the readOnly=true prop. Setting it to false prevents it from happening, but the editor is still non-editable regardless of that.


> but the editor is still non-editable regardless of that.

not sure about that 😅 maybe in the full environment, but in the test it definitely seems editable